department-of-veterans-affairs/caseflow
# Download hearing schedule

Connects #5953 

### Description

This PR adds functionality to download the current view on the View Schedule page

### Testing Plan

1) Navigate to the View Schedule page
1) Click 'Download Current View'
1) Confirm the downloaded csv is formatted correctly
